test/constant_folding.cpp

```diff
@@ -2376,3 +2376,73 @@ TEST(constant_folding, pass_property)
     ASSERT_FALSE(pass->get_property(pass::PassProperty::REQUIRE_STATIC_SHAPE));
     ASSERT_TRUE(pass->get_property(pass::PassProperty::CHANGE_DYNAMIC_STATE));
 }
+
+TEST(constant_folding, constant_non_zero_1D)
```

Comment:
Please, add a test case for 0D input.

Reply:
It was already tested via an onnx model but I've added a test here too.